src/braket/circuits/circuit.py

```diff
@@ -438,6 +440,134 @@ def add_circuit(
 
         return self
 
+    def add_noise(
+        self,
+        noise: Noise,
+        target_gates: Union[str, Iterable[str]] = None,
+        target_qubits: QubitSetInput = None,
+        target_times: Union[int, Iterable[int]] = None,
+        insert_strategy: str = "inclusive",
+    ) -> Circuit:
+        """
+        For any parameter that is None, that specification is ignored (e.g. if 'target_gates'
```

Comment:
Add an introductory paragraph that just describes what this method does (adding noise, and what that means)